### ISSUE TITLE
TypeError: Cannot read property 'model' of undefined

### ISSUE THREAD

**Issue by slavikme** [cellcortex/homebridge-yeelighter#57]
After upgrading to the latest version (v2.3.4), I have started receiving these errors:
```
TypeError: Cannot read property 'model' of undefined
    at YeelighterPlatform.configureAccessory (/homebridge/node_modules/homebridge-yeelighter/src/platform.ts:48:103)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:307:25
    at Array.filter (<anonymous>)
    at Server.restoreCachedPlatformAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:270:69)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:158:10)
    at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:80:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
```
I have tried to re-install the plugin but didn't help.

My config file is:
```js
{
   ...
   "accessories": [
       ...
       {
            "name": "Bedroom Light",
            "timeout": 5000,
            "interval": 60000,
            "ctforcolor": false,
            "blocking": false,
            "split": false,
            "platform": "Yeelighter"
        }
   ]
}
```

Any ideas what is wrong?

**Comment by cellcortex** [comment]
sorry - fixed in 2.3.5

**Comment by slavikme** [comment]
Works perfectly now. Thank you!
